1-exercises/J-string-substring/exercise3.js

```diff
@@ -8,12 +8,12 @@
 
 let statement = "I do not like programming";
 
-let result = "";
-
+// let result = "";
+let result = statement.substring(0, 4) + statement.substring(8);
```

Comment:
This works well. If you wanted to reuse the method to remove "not" from different sentences, you could find the index of "not".